### DISCUSSION TITLE
TailwindCSS classes like bg-yellow-300 are not picked up when I build

### DISCUSSION THREAD

**Discussion by man0l** [filamentphp/filament#9305] [Help] [answered]
Hello, 
I am making a widget which is positioned into the filament resources directory:

![image](https://github.com/filamentphp/filament/assets/1413216/b302ff50-8819-4afa-bc07-a738eb12b121)

This is a part of my blade template:

`        <div class="w-2/4 h-5 mx-4 bg-gray-200 rounded dark:bg-gray-700">
            <div class="h-5 bg-yellow-300 rounded" style="width: 70%"></div>
        </div>
        <span class="text-sm font-medium text-gray-500 dark:text-gray-400">{{ ($totalWithRating = $this->getTotalWithRaiting()) > 0 ? round($this->getStars(5) * 100 / $totalWithRating) . '%' : '0%' }}</span>
`

This is my tailwindcss config:

`import preset from './vendor/filament/support/tailwind.config.preset'

/** @type {import('tailwindcss').Config} */
export default {
  presets: [preset],
  content: [
    './app/Filament/**/*.php',
    './resources/views/filament/**/*.blade.php',
    './vendor/filament/**/*.blade.php',
  ],
  theme: {
    extend: {},
  },
  plugins: [],S
}
`

and this is my

`import { defineConfig } from 'vite';
import laravel, { refreshPaths } from 'laravel-vite-plugin';
import eslintPlugin from 'vite-plugin-eslint';

export default defineConfig({
  server: {
    host: true,
    port: 5173,
    watch: {
      usePolling: true
    },
    hmr: {
      host: 'localhost',
    },
  },
  plugins: [
    laravel({
      hotFile: 'public/hot',
      buildDirectory: 'app',
      input: [
        'resources/js/app.js',
        'resources/css/app.css',
        'resources/js/admin.js',
        'resources/css/admin.css'
      ],
      refresh: true,
    }),
    eslintPlugin({
      cache: false,
      include: ['resources/js/**/*.js'],
      exclude: ['node_modules/**', 'resources/js/admin.js'],
    }),
  ],
});`

I am runninng both npm run dev and npm run build and still the class does not exists in the built css in /public.
I don't have a theme registered.

Could you help me out?

**Comment by leandrocfe** [comment]
Are you using v3?
https://filamentphp.com/docs/3.x/panels/themes/

**Reply by man0l** [reply]
Yes, I use v3 and I do not have theme. Is this the issue?

**Reply by leandrocfe** [reply]
If you want to customize the css and add tailwind classes, you should follow this section

**Reply by man0l** [reply]
i don't want to customize. I want to use the utility classes but they're not in the built css. That's my question :)

**Reply by sinnbeck** [reply, marked as answer]
Using tailwind classes that are not already in use, require adding a theme :) No need to do anything special with that theme. Just extend the default and add it to the panel, and then run `npm run build` (or `npm run dev`)
